### PR TITLE
Add meta.yaml for pymc3-hmm conda forge

### PR DESCRIPTION
**Pull request by fanshi118**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by brandonwillard** [comment]
I'm willing to be listed here.

**Comment by dgerlanc** [comment]
I'm willing to be listed here

File: recipes/pymc3_hmm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
```
Comment (brandonwillard) [inline recipes/pymc3_hmm/meta.yaml:24]: Looks like this needs `pymc3`, at least.

**Review by brandonwillard — CHANGES_REQUESTED** [review]

File: recipes/pymc3_hmm/meta.yaml (hunk 1 above)
Comment (fanshi118) [reply recipes/pymc3_hmm/meta.yaml]: Updated. However, been getting [this dependency error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=279970&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=755) while running docker build. I basically copied the version dependencies directly from the `pymc3-hmm` repo. Do we need to adjust some of these?

**Comment by fanshi118** [comment]
@conda-forge-admin , please review the above, looks feedstock-ready 🙏

File: recipes/pymc3_hmm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
[the remark this answers is not in the document]
Comment (fanshi118) [reply recipes/pymc3_hmm/meta.yaml]: Yep, updated.

**Pull request merged**